tests/test_actions.py

```diff
@@ -453,6 +481,12 @@ def test_passes_if_all_pass(self, Tester):
             (True, IsEqualTo(True)),
         ).perform_as(Tester)
 
+    def test_describe(self):
+        tests = ((True, IsEqualTo(True)),
+                 (True, IsEqualTo(True)),
+                 (True, IsEqualTo(True)),
+                 (True, IsEqualTo(True)),)
```

Comment:
I'm surprised `black` is cool with this formatting, but then again i think i have `black` skip the tests files...?

Reply:
Black skips tests. that goes into the larger question of should pre-commit hooks look at tests?